sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+* They may grow a set of practices over time in the same way as an OSS Project, provided this is consistent with CNCF Operating Principles. 
+
+* As with CNCF Projects all exceptions and disputes are handled by TOC with CNCF Staff help
+
+### Budget & Resource
+
+* No formal systematic budget at this time, other than commitment of CNCF executive staff to provide named person as liaison point.
+
+* Just as CNCF Projects may have "help" offered by CNCF and may ask for things via the [ServiceDesk](https://github.com/cncf/servicedesk), the SIGs may do this.
+
+## Retirement
+
+* In the event that a SIG is unable to regularly establish quorum, or fulfill the responsibilities and/or regularly report to the TOC, the TOC will:
+
+    * Consider retiring the SIG after 3 months
+
+    * Must retire the SIG after 6 months
+
+* The TOC may, by means of a 2/3 majority vote, declare "no confidence" in the SIG.  In this event, the TOC may then vote to retire or reconstitute the SIG.
+
+## Initial SIGS
+
+To bootstrap the process, the TOC proposes the following SIGs, and projects assigned to each SIG. Clearly all of these SIG’s will not be fully-formed overnight or begin operating immediately, so the TOC itself will fulfill the duties of not-yet-formed SIG’s until they are.  We can however, fairly immediately, assign one voting member of the TOC as liason for each SIG, and prioritize the order of formation of the SIGs, starting immediately with the most pressing ones. 
+
+<table>
+  <tr>
+    <td>Name (to be finalised)</td>
+    <td>Area</td>
+    <td>Current CNCF Projects</td>
+  </tr>
+  <tr>
+    <td>Traffic</td>
+    <td>networking, service discovery, load balancing, service mesh, RPC, pubsub, etc.</td>
+    <td>Envoy, Linkerd, NATS, gRPC, CoreDNS, CNI</td>
+  </tr>
+  <tr>
+    <td>Observability</td>
+    <td>monitoring, logging, tracing, profiling, etc.
+</td>
+    <td>Prometheus, OpenTracing, Fluentd, Jaeger, Cortex, OpenMetrics, </td>
+  </tr>
+  <tr>
+    <td>Governance</td>
+    <td>security, authentication, authorization, auditing, policy enforcement, compliance, GDPR, cost management, etc</td>
+    <td>SPIFFE, SPIRE, Open Policy Agent, Notary, TUF,  Falco, </td>
+  </tr>
+  <tr>
+    <td>App Dev, Ops & Testing</td>
+    <td>PaaS, Serverless, Operators,... CI/CD,  Conformance, Chaos Eng, Scalability and Reliability measurement etc.</td>
+    <td>Helm, CloudEvents, Telepresence, Buildpacks, (CNCF CI)</td>
+  </tr>
+  <tr>
+    <td>Core and Applied Architectures</td>
+    <td>orchestration, scheduling, container runtimes, sandboxing technologies, packaging and distribution, specialized architectures thereof (e.g. Edge, IoT, Big Data, AI/ML, etc).</td>
+    <td>Kubernetes, containerd, rkt, Harbor, Dragonfly, Virtual Kubelet</td>
+  </tr>
+  <tr>
+    <td>Storage</td>
+    <td>Block and File Stores, Databases, Key-Value stores etc.</td>
```

Comment:
Can this include *Object* next to *Block* and *File*?

Reply:
Absolutely! Good catch.  Done.